src/django/pfb_analysis/views.py

```diff
@@ -12,24 +12,26 @@
 from rest_framework.viewsets import ModelViewSet
 from rest_framework.response import Response
 
-from pfb_analysis.models import AnalysisJob, Neighborhood
-from pfb_analysis.serializers import AnalysisJobSerializer, NeighborhoodSerializer
 from pfb_network_connectivity.pagination import OptionalLimitOffsetPagination
-from pfb_network_connectivity.filters import (OrgAutoFilterBackend,
-                                              SelfUserAutoFilterBackend,
-                                              AnalysisJobStatusFilterSet)
+from pfb_network_connectivity.filters import OrgAutoFilterBackend
 from pfb_network_connectivity.permissions import IsAdminOrgAndAdminCreateEditOnly, RestrictedCreate
 
+from .models import AnalysisJob, Neighborhood
+from .serializers import AnalysisJobSerializer, NeighborhoodSerializer
+from .filters import AnalysisJobFilterSet
+from .functions import ObjectAtPath
+
 
 class AnalysisJobViewSet(ModelViewSet):
     """For listing or retrieving analysis jobs."""
 
-    queryset = AnalysisJob.objects.all()
+    queryset = AnalysisJob.objects.all().annotate(
```

Comment:
Presumably this annotation is so that the sorting works, right?

This annotation is presumably useful elsewhere. Consider moving the logic to a custom AnalysisJobManager in models.py.